src/components/App.jsx

```diff
@@ -1,47 +1,51 @@
+/* eslint-disable import/extensions */
 import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
 import Ratings from './Ratings.jsx';
 import ReviewList from './ReviewList.jsx';
-import {exampleReview, exampleRating} from './exampleReview.js';
-import axios from 'axios';
-
-
-
-
-
-function App(props) {
+import { exampleReview, exampleRating } from './exampleReview.js';
 
+function App() {
+  const initialState = exampleRating;
   const [reviews, setReviews] = useState([]);
-  const [ratings, setRatings] = useState(exampleRating);
+  const [ratings, setRatings] = useState(initialState);
+  const [error, setError] = useState(null);
+  const [reviewsLoaded, setReviewsLoaded] = useState(false);
+  const [ratingsLoaded, setRatingsLoaded] = useState(false);
 
   useEffect(() => {
     axios.get('http://52.26.193.201:3000/reviews/1/list')
-  .then(response => {
-    setReviews(response.data.results);
+      .then((response) => {
+        setReviewsLoaded(true);
+        setReviews(response.data.results);
+      });
+    axios.get('http://52.26.193.201:3000/reviews/4/meta')
+      .then((response) => {
+        setRatingsLoaded(true);
+        setRatings(response.data);
+      });
+  }, [reviews]);
+
+  if (!reviewsLoaded || !ratingsLoaded) {
+    return <div>Loading...</div>;
   }
```

Comment:
I'm not sure it is necessary to have this loading functionality.  does it take that long to perform the get request?  I'd imagine it's fast enough that you would never see the loading div.